src/main/groovy/com/miniasaaslw/adapters/emailnotification/EmailNotificationAdapter.groovy

```diff
@@ -19,7 +21,7 @@ class EmailNotificationAdapter {
     public EmailNotificationAdapter buildPayerEmailPaymentCreated(Payment payment) {
         this.recipientEmail = payment.payer.email
         this.subject = MessageUtils.getMessage("emailNotification.subject.payment.created")
-        this.url = "/payment/checkout/${payment.publicId}"
+        this.url = BASE_URL + "/payment/checkout/${payment.publicId}"
```

Comment:
Não é melhor colocar a baseUrl na hora que for renderizar em tela?
Se o servidor muda de URL, todas as notificações param de funcionar